docs/files/integration.rst

```diff
@@ -0,0 +1,162 @@
+your project and with your virtual environment selected you can
+
+.. code-block:: console
+
+   $ pip install invenio[auth]
+
+
+Files bundle
+------------
+
+As mentioned before functionality for files support is broken down in different
+parts to provide as much flexibility as possible.
+
+`invenio-files-rest <https://invenio-files-rest.readthedocs.io/>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+   $ pip install invenio-files-rest
+
+:code:`invenio-files-rest` is the first required module for managing files
+with your application, and some of its key functions is to allow you to store
+and retrieve files in a similar way to Amazon S3 APIs.
+
+- Configurable files storage
+- Secure REST APIs for upload/download
+- Support for large file uploads and multipart upload.
+- File integrity monitoring
+- Customizable access control
+
+After you install the module on your environment you just have to register
+the invenio-files-rest blueprint in the list of blueprints defined in the
+:code:`entry_points` of the :code:`setup.py` of your application.
+
+.. code-block:: python
+
+    entry_points={
+        "invenio_app.api_blueprints": [
+            "invenio_files = invenio_files_rest.views:blueprint",
+        ],
+    }
+
+Read more about `configuration <https://invenio-files-rest.readthedocs.io/en/latest/configuration.html>`__
+and `usage <https://invenio-files-rest.readthedocs.io/en/latest/usage.html>`__
+of invenio-files-rest on the module's `documentation <https://invenio-files-rest.readthedocs.io/>`__.
+
+
+`invenio-records-files <https://invenio-records-files.readthedocs.io/>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: console
+
+   $ pip install invenio-records-files
+
+:code:`invenio-records-files` is the other required module, which provides a
+basic API for the seamless co-operation of `invenio-records <https://invenio-records.readthedocs.io/>`__
+and `invenio-files-rest`_. The API provides functionality for
+
+- records creation
+- files creation
+- accessing files
+- files metadata management
+- files extraction from records
+
+After installing the module you will need to provide the relevant
+configuration in the :code:`config.py` of your application, for the
+endpoints you want to use, it should be in the following form
+
+.. code-block:: python
+
+    RECORDS_FILES_REST_ENDPOINTS = {
+        '<*_REST_ENDPOINTS>': {
+            '<endpoint-prefix>': '<endpoint-suffix>',
+        }
+    }
+
+Read more about `configuration <https://invenio-records-files.readthedocs.io/en/latest/configuration.html>`__
+and `usage <https://invenio-records-files.readthedocs.io/en/latest/usage.html>`__
+of invenio-records-files on the module's `documentation <https://invenio-records-files.readthedocs.io/>`__.
+
+
+`invenio-previewer <https://invenio-previewer.readthedocs.io/>`_
```

Comment:
Seems like we have worked on the same issue here. Perhaps we could merge what we wrote in one place?